src/content/guides/prepare-azure-subscription-for-guest-clusters/index.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Prepare an Azure subscription to run Giant Swarm guest clusters"
+description = "This guide will walk you through all necessary steps to set up an Azure subscription with approriate Role definition and Service Principal for operating Giant Swarm guest clusters."
+date = "2018-08-09"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+
+# Prepare an Azure subscription to run Giant Swarm guest clusters
+
+In a Giant Swarm installation the guest clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the host cluster. This gives greater flexibility depending on the requirements and the use case. For example, it allows the host cluster to be running in one Azure subscription, while guest clusters operate in different Azure subscriptions, depending on a customer’s department using them.
+
+Giant Swarm operates guest clusters using a service called `azure-operator` which runs on the host cluster.
+
+## Overview
+
+In order to run Giant Swarm guest clusters, an Azure subscription needs the following elements:
+
+- Role definition: a set of permission to operate guest clusters in the Azure subscription.
+- Service Principal: an identity (bound to the previously defined Role definition) to access the Azure subscription.
+
+## Create Azure Role definition and Service Principal
+
+In order to perform necessary actions to deploy and maintain guest clusters in your Azure subscription, `azure-operator` needs to access the subscription using a Service Principal.
+Here we describe all the steps to set it up.
+
+#### 1. Prerequisites
+
+To create a Service Principal you need:
+
+- An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](User Access Administrator) role.
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed.
+
+#### 2. Role definition
+
+Download our [Role definition template](https://raw.githubusercontent.com/giantswarm/azure-operator/master/policies/guest.tmpl.json).
```

Comment:
We are no longer telling the user to replace the subscription ID in the template file.

Reply:
fixed